internal/xcrd/crd_test.go

```diff
@@ -402,155 +409,19 @@ func TestForCompositeResource(t *testing.T) {
 	}
 }
 
-func TestValidateClaimNames(t *testing.T) {
```

Comment:
The way git decided to parse this diff is rather unfortunate. I didn't change anything in `TestValidateClaimNames` except perhaps what the go linting tool did automatically. 

I validated that the only changes in d793208b were the addition of the two `Test...EmptyXrd` functions by deleting them locally and observing an empty git diff with its parent commit.